Render/RenderSettings.as

```diff
@@ -43,6 +43,11 @@ void RenderSettingsCustomization(){
         hiddingSpeedSetting = 0;
     }
 
+    unhideDelay = UI::InputInt("(ms) Unhide delay (if the hide when driving mode is active)", unhideDelay, 100);
```

Comment:
May I suggest "Delay (in ms) to wait before showing the window if you're below the hiding speed"? You can keep the variable name as-is though